package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-pr-calculator",
-  "version": "1.16.14",
+  "version": "1.16.15",
```

Comment:
Could you please upgrade 'govuk-frontend : 4.3.1' module  in line 37 to 'govuk-frontend: 4.7.0'